app/models/restaurant.rb

```diff
@@ -8,4 +8,20 @@ class Restaurant < ActiveRecord::Base
   validates :city, presence: true
   validates :state, presence: true
   validates :zip_code, presence: true
+
+  def average_rating
+    self.reviews.sum(:rating)/self.reviews.count
```

Comment:
Cool - just wanted to check since Elliott and I ended up removing the calls to self on some of the voting methods